content/blog/entries/cc-search-accessibility-week5-6/contents.lr

```diff
@@ -0,0 +1,80 @@
+title: Internationalization continued: Modifying tests
+---
+categories:
+cc-search
+community
+gsoc
+open-source
+
+---
+author: AyanChoudhary
+---
+series: gsoc-2020-ccsearch-accessibility
+---
+pub_date: 2020-07-10
+---
+body:
+
+These are the fifth and sixth weeks of my internship with CC. I am working on improving the accessibility of cc-search and internationalizing it as well.
+This post contains yet another important aspect to be taken care of while internationalizing components, i.e. modifying tests to include the changes.
+
+The components which were left are the two pages displaying the most content:
+1. Browse page
```

Comment:
Go ahead and link these components to their files in the repo! A good way to cross-link to the project itself.